packages/sdk/src/api/converter.ts

```
@@ -1547,6 +1550,12 @@ function fromOperation(pbOperation: PbOperation): Operation | undefined {
       treeRestoreMode,
       treeRetombstoneSpans,
     );
+    treeEdit.setSplitTickets(
+      pbTreeEditOperation!.splitTickets.map(
+        (ticket) => fromTimeTicket(ticket)!,
```

ESLint

[error] Replace `⏎········(ticket)·=>` with `(ticket)·=>⏎·······`

(prettier/prettier)